Gemfile

```diff
@@ -4,16 +4,15 @@ source 'https://rubygems.org'
 
 gem 'cocoapods', '~> 1.11'
 gem 'commonmarker'
-gem 'danger', '~> 8.6'
+gem 'danger', '~> 9.3'
 gem 'danger-rubocop', '~> 0.10'
 gem 'dotenv'
 gem 'fastlane', '~> 2.174'
 gem 'fastlane-plugin-appcenter', '~> 1.8'
 gem 'fastlane-plugin-sentry'
 # This comment avoids typing to switch to a development version for testing.
 # gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit', branch: 'trunk'
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 7.0'
-gem 'octokit', '~> 4.0'
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 8.0', '>= 8.0.1'
```

Comment:
- Removed the explicit `octokit` dependency from `Gemfile` since it was unused and clashed with the new toolkit version.
- The update `octokit` requirement in the toolkit also clashed with Danger. The tool required a major version upgrade, which should be safe since the only breaking change was dropping Ruby 2.4. See
  https://github.com/danger/danger/blob/85a2a6f05f2a25fcf3fad4fcf65360186f16192f/CHANGELOG.md

Reply:
@mokagio Sorry about those `octokit` and `danger` issues. I marked the `octokit` upgrade as a non-breaking change because I didn't think it would change any client behavior. I didn't realize that some clients use either of those gems directly.